pages/post.tsx

```diff
@@ -103,103 +151,142 @@ const post = () => {
+  }
+
+  const handleEndTimer = (newValue: Date) => {
+    setEndAt(newValue)
+  }
+
   // step별 컴포넌트 로직
   const getStepContent = (stepNumber: number) => {
     switch (stepNumber) {
       case 0:
         return (
           <EventTitle
-            eventTitle={eventTitle}
-            participant={participant}
-            coverImage={coverImage}
+            title={title}
+            maxParticipantCount={maxParticipantCount}
+            mainTemplate={mainTemplate}
             handleInput={handleInput}
             handleCoverImage={handleCoverImage}
           />
         )
       case 1:
-        return (
-          <EventTimer
-            startvalue={startvalue}
-            endvalue={endvalue}
-            handleStartTimer={handleStartTimer}
-            handleEndTimer={handleEndTimer}
-          />
-        )
-      case 2:
         return (
           <EventType
-            eventTypeState={eventTypeState}
+            giftChoiceType={giftChoiceType}
             handleTypeCheck={handleTypeCheck}
           />
         )
-      case 3:
+      case 2:
         return (
           <EventPresent
             gifts={gifts}
             AddGiftItem={AddGiftItem}
             delteGiftItem={delteGiftItem}
           />
         )
+      case 3:
+        return (
+          <EventTimer
+            startAt={startAt}
+            endAt={endAt}
+            handleStartTimer={handleStartTimer}
+            handleEndTimer={handleEndTimer}
+          />
+        )
       default:
         return
     }
   }
 
   return (
-    <Box sx={{ width: '100%' }}>
-      <Stepper activeStep={activeStep} alternativeLabel>
-        {steps.map((label, index) => {
-          const stepProps: { completed?: boolean } = {}
-          const labelProps: {
-            optional?: React.ReactNode
-          } = {}
-
-          return (
-            <Step key={label} {...stepProps}>
-              <StepLabel {...labelProps}>
-                <Text size={'MICRO'} color={'WHITE'}>
-                  {label}
-                </Text>
-              </StepLabel>
-            </Step>
-          )
-        })}
-      </Stepper>
-
-      <div style={{ color: 'white' }}>{getStepContent(activeStep)}</div>
-
-      <StepButtonContainer>
-        <DisplayStyle activeStep={activeStep}>
-          <MUIButton
-            style={{
-              color: '#ffffff',
-              backgroundColor: '#000000',
-            }}
-            onClick={handleBack}
-            sx={{ mr: 1 }}>
-            Back
-          </MUIButton>
-        </DisplayStyle>
-
-        <MUIButton
-          style={{ color: '#ffffff', backgroundColor: '#CE000B' }}
-          onClick={handleNext}>
-          {activeStep === steps.length - 1 ? 'Finish' : 'Next'}
-        </MUIButton>
-      </StepButtonContainer>
-    </Box>
+    <>
+      {activeStep === steps.length ? (
+        <EventComplete
+          eventLink={eventLink}
+          giftChoiceType={giftChoiceType}></EventComplete>
```

Comment:
```suggestion
        <EventComplete
          eventLink={eventLink}
          giftChoiceType={giftChoiceType} />
```
별도 Children을 받지 않는다면 항상 닫는 태그가 컨벤션인 것으로 알고있습니다!